packages/kit/src/core/sync/write_types/test/simple-page-server-only/_expected/$types.d.ts

```diff
@@ -20,11 +20,15 @@ export type PageServerLoad<
 	OutputData extends OutputDataShape<PageServerParentData> = OutputDataShape<PageServerParentData>
 > = Kit.ServerLoad<RouteParams, PageServerParentData, OutputData>;
 export type PageServerLoadEvent = Parameters<PageServerLoad>[0];
-export type Errors = null;
+export type ActionData = Kit.AwaitedActions<
+	typeof import('../../../../../../../../+page.server.js').actions
```

Comment:
i don't really understand why there are so many `../` segments, but if the tests pass then i'm gonna pretend i didn't see it 🙈

Reply:
It's because the actual file that is compared against is deeply nested; I didn't want to create 5 useless folders in `_expected` 😄